web_src/css/helpers.css

```diff
@@ -83,14 +83,17 @@ Gitea's private styles use `g-` prefix.
 .gt-w-screen { width: 100vw !important; }
 .gt-h-screen { height: 100vh !important; }
 
+.gt-min-h-0 { min-height: 0 !important; }
+.gt-min-w-0 { min-width: 0 !important; }
+
```

Comment:
```suggestion
```

It seems that these two helpers are not used now, and maybe they should only appear with other styles, so I think there is no need to add them as helpers.

Reply:
We will have to re-add them at some point, but whatever you prefer.